packages/@pollyjs/adapter/src/index.js

```diff
@@ -229,40 +229,61 @@ export default class Adapter {
 
   /**
    * @param {PollyRequest} pollyRequest
-   * @returns {*}
+   * @returns {response}
    */
-  onRecord() {
-    this.assert('Must implement the `onRecord` hook.', false);
+  async passthroughRequest(/* pollyRequest */) {
+    this.assert('Must implement `passthroughRequest`.', false);
+  }
+
+  /* Other Hooks */
+  /**
+   * @param {PollyRequest} pollyRequest
+   * @returns {PollyRequest}
+   */
+  async onRecord(pollyRequest) {
+    await this.onPassthrough(pollyRequest);
+    await this.persister.recordRequest(pollyRequest);
+
+    return pollyRequest;
```

Comment:
We can simplify these hooks even more by removing the need to return the pollyRequest and just have `handleRequest` return it instead as it already has it.

Reply:
Done